docs/react/README.md

```diff
@@ -1,72 +1,186 @@
+### createJsonapiClient
 
-### TypeScript
+`createJsonapiClient(baseUrl: string, schemaMap: Schemas)` is a constructor function that creates a new JSON:API rest client from the defined schemas. It accepts a base url, and schema set. For more documentation see [here](./rest-client.md) 🛑.
 
-@hatchifyjs/react includes TypeScript support. Here's an example of how two schemas (`Todo` and `User`) provide auto-completion for an instantiated @hatchifyjs/react app (`hatchedReact`):
+```ts
+import {createJsonapiClient} from "@hatchifyjs/react"
 
-![react TypeScript](../attachments/reactTs.gif)
+const schemas = { ... }
 
-## Types
+const jsonClientInstance = createJsonapiClient("/api", Schemas)
+```
 
-There are a number of types available to assist with app customization.
+**Parameters**
+`createJsonapiClient` takes two arguments `baseUrl` and `schemaMap`
+`baseUrl` is a `string` that references the base url for the rest client
+`schemaMap` is a collectionn of [Hatchify Schemas](../schema/README.md)
 
-- DataGridState
-- CreateType
-- UpdateType
-- HatchifyApp
-- RecordType
+**Returns**
 
-Learn more about the available types [here](./types.md).
+Returns a `JSON:API rest client` instance object
 
-## createJsonapiClient
+### hatchifyReact
 
-`createJsonapiClient` creates a new JSON:API rest client from the defined schemas. It accepts a base url, and schema set. For more documentation see [here](./rest-client.md) 🛑.
+`hatchifyReact(createJsonapiClient("/api", Schemas))` is a `Function` that initializes the `HatchifyApp` object from the JSON:API rest client. Inside of the retured object you will find [`components`](./components.md), [`model`](./model.md), [`state`](./state.md), and [`Everything`](#everything).
 
 ```ts
+import {createJsonapiClient, hatchifyReact} from "@hatchifyjs/react"
+
+const schemas = { ... }
+
 const hatchedReact = hatchifyReact(createJsonapiClient("/api", Schemas))
 ```
 
-## HatchifyProvider
+### HatchifyProvider
 
-`HatchifyProvider` is a component that hosts and provides access to Hatchify-related state. Hatchify components in `HatchifyProvider`'s subtree interact with it to access Hatchify internal state. It must be a parent to any Hatchify components.
+`HatchifyProvider` is a component that hosts and provides access to Hatchify-related state. It must be a parent to any Hatchify components.
 
 ```tsx
-const App: React.FC = () => {
+import {HatchifyProvider} from "@hatchifyjs/react"
+
+// Define Schemas
+
+const schemas = { ... }
+
+// Create the Hatched React App instance
+const hatchedReact = hatchifyReact(createJsonapiClient("/api", Schemas))
+
+// Define variables for your Hatchify schema generated components
+const TodoDataGrid = hatchedReact.components.Todo.DataGrid
+
+const HatchedComponent = () => {
```

Comment:
same stuff in this snippet and others - should just be `function App()` everywhere, it needs the mui imports for provider and wrap it.

Reply:
this code snippet is almost the same as the one at the top of the page - maybe we don't need it?